### PR TITLE
Remove bootstrap v3

### LINKED ISSUE #23526: [Identity] Remove bootstrap 3 support from Identity UI
Bootstrap 3 is out of support, so we should rely only on more modern bootstrap versions for future releases.

### PR DESCRIPTION
**Pull request by HaoK**
Fixes https://github.com/dotnet/aspnetcore/issues/23526

### CODE REVIEW

File: src/Identity/UI/src/build/Microsoft.AspnetCore.Identity.UI.targets [hunk 1]
```diff
@@ -3,26 +3,14 @@
...
-             Condition="'$(IdentityUIFrameworkVersion)' == 'Bootstrap3' and '%(_IdentityUIActiveWarnings.Identity)' == 'IdentityUI001' and '$(WarningsAsErrors)' != 'true'" />
-
-    <Error Code="IdentityUI001"
-           Text="$(_IdentityUIBootstrap3WarningMessage)"
-           File="@(MSBuildThisProjectFile)"
-           Condition="'$(IdentityUIFrameworkVersion)' == 'Bootstrap3' and '%(_IdentityUIActiveWarnings.Identity)' == 'IdentityUI001' and '$(WarningsAsErrors)' == 'true'" />
-
     <Error
            Code="IdentityUI002"
            Text="$(_IdentityUIUnknownVersionErrorMessage)"
```
Comment (javiercn) [inline src/Identity/UI/src/build/Microsoft.AspnetCore.Identity.UI.targets:16]: Line 18 needs updating too

**Comment by javiercn** [comment]
Looks good, I suspect there are a bunch of tests that need to be removed too?

**Comment by HaoK** [comment]
Tests are passing again, how's it look @javiercn

**Review by javiercn — APPROVED** [review]
Looks great!

Thanks for taking care of this!

**Comment by HaoK** [comment]
@deepchoudhery we've removed the V3 bootstrap from the UI for 5.0, can you help me identify how to do the equivalent for scaffolding?  Is it as simple as nuking the V3 files, or is there more stuff we need to delete?

**Comment by deepchoudhery** [comment]
> @deepchoudhery we've removed the V3 bootstrap from the UI for 5.0, can you help me identify how to do the equivalent for scaffolding? Is it as simple as nuking the V3 files, or is there more stuff we need to delete?

The Bootstrap3 folder [here ](https://github.com/dotnet/Scaffolding/tree/master/src/VS.Web.CG.Mvc/Templates/Identity) should be all of it. There are some Bootstrap checks we'd remove as well. If you want, I can do this for master(consequent 5.0). Is this change for 5.0-Preview7 or Preview8?

**Comment by HaoK** [comment]
Preview8, thanks @deepchoudhery !

**Pull request merged**